.env.example

```
@@ -2,4 +2,7 @@
 DATABASE_URL=
 
 # NEXT
-NEXT_PUBLIC_APP_URL=http://localhost:3000 # change this with prod url 
\ No newline at end of file
+NEXT_PUBLIC_APP_URL=http://localhost:3000 # change this with prod url 
+
+# OPENAI
+OPENAI_API_KEY=
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)